containers/init/clone/clone.sh

```diff
@@ -16,9 +16,27 @@
 set -ex
 cd /src/workspace
 ls -A | xargs -r rm -fr
+
+# The git clone command clones code off the master branch by default or a branch
```

Comment:
I think the explanation becomes clearer if it is limited to what is actually happening, and points to where the idea came from. Something like this (deleting the first paragraph completely – additional discussion might go to the description of the commit of PR, if useful):

```shell
# This process initializes an empty git repository, adds and fetches objects
# from the $CLONE_REPO, checks out the $CLONE_GIT_REF and then updates the
# submodules. This prevents the unnecessary checkout of the master branch.
# This process is similar to other CI systems, including GitHub actions. See:
# https://stackoverflow.com/questions/3489173.
```
I am not even sure about keeping the mention of "other CI systems", since there is no reference pointing to that, but it's ok to keep.

Reply:
Ping (1 month).